Enterprise-CMCS/macpro-mdct-qmr
# IET Total Rates / Validation

### Description

For measure IET-AD & IET-HH, there's a new way of calculating total's in the N/D/R set. 

Normally, total summations are done in the qualifier layer, where the last qualifier is assumed to be the area for capturing the total data. 

In IET-AD & IET-HH, we want to capture data from the category layer, where the summation of the category's qualifiers is the sum of all the previous qualifiers of the same type (i.e. Ages 18 to 64 is the sum of all the previous Ages 18 to 64 in the other categories)

So it is sorted by categories: **Initiation of SUD Treatment** & **Engagement of SUD Treatment**.
Then further sorted by qualifiers: 
- Ages 18 to 64
- Age 65 and older


### Related ticket(s)


MDCT-2452

---
### How to test

1) Sign into QMR, any account
2) Select Reporting Year 2023
3) Go to IET-AD measure
4) Select first radio for first 3 questions
5) In N/D/R field, enter values in any category but the total categories ( Initiation of SUD Treatment: Total & Engagement of SUD Treatment: Total). 
6) Qualifiers in the Total categories should be the summation of all the previous non total categories.

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
